app/components/file_upload_preview/file_upload_preview.js

```diff
@@ -46,17 +48,30 @@ export default class FileUploadPreview extends PureComponent {
 
     buildFilePreviews = () => {
         return this.props.files.map((file) => {
+            let filePreviewComponent;
+            if (file.loading | file.has_preview_image) {
```

Comment:
Yes, if the file is loading the `<FileAttachmentImage/>` component will display the loading spinner.